help/develop/onboarding/exercises.md

```diff
@@ -623,14 +623,44 @@ First, let's explore how scripting viewer works.
    `Data` corresponds to the first button from the top of the Datagrok sidebar. Make sure the table view with the data
    appears.
 1. Activate the top menu from the sidebar, using a `Windows | Menu` switch. 3. In this menu, hit `Add | Scripting
-Viewers | Python | Scatter Plot`.
+Viewers | Add Scripting
+Viewer`.
 1. See that the viewer appeared on the right, telling though it is "Unable to plot with current settings".
 1. Proceed to the viewer properties by hitting on the gear icon in the viewer's title.
 1. Make sure the chosen values for "Data" are `HEIGHT` for `X`, `WEIGHT` for `Y`, and `AGE` for `Color`. After checking
    this you should see a nice scatter plot for `WEIGHT` and `HEIGHT` with the color corresponding to `AGE`:
    ![exercises-scripting-viewer](exercises-scripting-viewer.png)
 1. In the context panel, proceed to modify the value of the "Script" field by clicking on a "..." icon in the text
    field.
+1. Enter next code to script editor:
```

Comment:
please look one more time through the text you've written with Grammarly or ChatGPT
this sentence should be like: "Enter the next code into the script editor"